src/main/java/com/github/instagram4j/instagram4j/requests/usertags/UserTagsRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.github.instagram4j.instagram4j.requests.usertags;
+
+
+import com.github.instagram4j.instagram4j.requests.IGGetRequest;
+import com.github.instagram4j.instagram4j.responses.usertags.UserTagsResponse;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class UserTagsRequest extends IGGetRequest<UserTagsResponse> {
```

Comment:
This seems to be a feed request which might have pagination (a next max id in the response). If so, please make this a paginated request and have a paginated response.

Examples are here:
[FeedTimelineRequest](https://github.com/instagram4j/instagram4j/blob/b491daa700d38a7dc13725400cff8012723d5d79/src/main/java/com/github/instagram4j/instagram4j/requests/feed/FeedTimelineRequest.java#L19)
[FeedTimelineResponse](https://github.com/instagram4j/instagram4j/blob/b491daa700d38a7dc13725400cff8012723d5d79/src/main/java/com/github/instagram4j/instagram4j/responses/feed/FeedTimelineResponse.java#L15)